android/app/src/main/java/org/fossasia/openevent/data/Event.java

```diff
@@ -23,356 +14,435 @@
+    @JsonProperty("can-pay-by-stripe")
+    private boolean canPayByStripe;
+    @JsonProperty("large-image-url")
+    private String largeImageUrl;
+    @JsonProperty("timezone")
     private String timezone;
-    private String logo;
-    private String searchableLocationName;
-    private String large;
-    private CallForPapers callForPapers;
+    @JsonProperty("can-pay-onsite")
+    private boolean canPayOnsite;
+    @JsonProperty("deleted-at")
+    private String deletedAt;
+    @JsonProperty("ticket-url")
+    private String ticketUrl;
+    @JsonProperty("can-pay-by-paypal")
+    private boolean canPayByPaypal;
+    @JsonProperty("location-name")
     private String locationName;
-    private String name;
-    private Copyright copyright;
+    @JsonProperty("is-sponsors-enabled")
+    private boolean isSponsersEnabled;
+    @JsonProperty("is-sessions-speakers-enabled")
+    private boolean isSessionsSpeakersEnabled;
+    @JsonProperty("privacy")
     private String privacy;
-    private String placeholderUrl;
-    @Relationship("social-links")
-    private RealmList<SocialLink> socialLinks;
-    private Double longitude;
-    private String organizerName;
-    private String schedulePublishedOn;
+    @JsonProperty("code-of-conduct")
+    private String codeOfConduct;
+    @JsonProperty("state")
     private String state;
-    private Version version;
-    private Boolean hasSessionSpeakers;
-    private String subTopic;
+    @JsonProperty("latitude")
     private Double latitude;
-    private String organizerDescription;
-    private String identifier;
-    private String type;
-    private String email;
-    private String codeOfConduct;
+    @JsonProperty("longitude")
+    private Double longitude;
+    @JsonProperty("starts-at")
+    private String startsAt;
+    @JsonProperty("ends-at")
+    private String endsAt;
+    @JsonProperty("searchable-location-name")
+    private String searchableLocationName;
+    @JsonProperty("is-ticketing-enabled")
+    private boolean isTicketingEnabled;
+    @JsonProperty("can-pay-by-cheque")
+    private boolean canPayByCheque;
+    @JsonProperty("description")
+    private String description;
+    @JsonProperty("pentabarf-url")
+    private String pentabarfUrl;
+    @JsonProperty("xcal-url")
+    private String xcalUrl;
+    @JsonProperty("event-url")
+    private String eventUrl;
+    @JsonProperty("logo-url")
+    private String logoUrl;
+    @JsonProperty("is-tax-enabled")
+    private boolean isTaxEnabled;
+    @JsonProperty("ical-url")
+    private String icalUrl;
+    @JsonProperty("icon-image-url")
+    private String iconImageUrl;
+    @JsonProperty("name")
+    private String name;
+    @JsonProperty("can-pay-by-bank")
+    private boolean canPayByBank;
+    @JsonProperty("created-at")
+    private String createdAt;
+    @JsonProperty("bank-details")
+    private String bankDetails;
+    @JsonProperty("cheque-details")
+    private String chequeDetails;
 
-    @JsonSetter("thumbnail")
```

Comment:
You may not support the new API at all but you have to support the json export format. That was the entire point of testing in opev-android-sdk